Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub_test.py

```diff
@@ -674,7 +674,7 @@ def test_try_pull_unique_messages__unique_first_try(self, mocker):
         ) = try_pull_unique_messages(
             client, sub_name, previous_msg_ids, last_run_time, retry_times=1
         )
-        assert debug_mock.call_count == 0
+        assert debug_mock.call_count == 1  # call in "extract_acks_and_msgs" function
```

Comment:
didnt we discuss asserting all calls arguments were not with the unexpected log? or that they were with the expected logs?

Reply:
ill do it